### PR TITLE
IGAPP-1256: Remove placeholder icon

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: release-notes/unreleased/IGAPP-1256-change-circle-icons.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+issue_key: IGAPP-1256
+show_in_stores: true
+platforms: 
+  - web
+  - ios
+  - android
+en: Remove placeholder icon in category lists.
+de: Entfernen des Platzhalter Symbols in der Kategorienlisten.
```
Comment (LeandraH) [inline release-notes/unreleased/IGAPP-1256-change-circle-icons.yml:8]:
```suggestion
de: Entfernen des Platzhalter-Symbols in den Kategorienlisten.
```

**Review by LeandraH — APPROVED** [review]
Nice! Tested in Firefox and simulated iOS

**Review by f1sh1918 — CHANGES_REQUESTED** [review]
I just checked the proposed design
https://www.figma.com/file/cA4F2MwHs2LNGviOWUjfE9/Integreat-Tickets-Jira?node-id=830-750&t=WKp8IRlXK40Dj8Rj-0
The border should take the whole space. I think it has to be discussed what happens with second lvl categories but currently it looks very weird in my opinion. Please request a complete list design from ui/ux.
image

**Comment by sarahsporck** [comment]
> I just checked the proposed design
> https://www.figma.com/file/cA4F2MwHs2LNGviOWUjfE9/Integreat-Tickets-Jira?node-id=830-750&t=WKp8IRlXK40Dj8Rj-0
> The border should take the whole space. I think it has to be discussed what happens with second lvl categories but currently it looks very weird in my opinion. Please request a complete list design from ui/ux.

👍🏼 werde mich nochmal mit toni dazu austauschen und dann nen neuen PR öffnen.

**Pull request closed without merging**